cardigan_new/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@weco/cardigan_new",
+  "license": "MIT",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "storybook dev -c .storybook",
+    "build": "storybook build -c .storybook -o .dist",
+    "chromatic": "chromatic --only-changed"
+  },
+  "dependencies": {
+    "@faker-js/faker": "^8.4.1",
+    "@storybook/addon-a11y": "^8.3.5",
+    "@storybook/addon-backgrounds": "^8.3.5",
+    "@storybook/addon-controls": "^8.3.5",
+    "@storybook/addon-docs": "^8.3.5",
+    "@storybook/cli": "^8.3.5",
+    "@storybook/nextjs": "^8.3.5",
+    "@storybook/react": "^8.3.5",
+    "@storybook/theming": "^8.3.5",
+    "chromatic": "^6.21.1",
+    "husky": "^4.3.0",
+    "next": "13.5.2",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "typescript": "^4.9.5",
```

Comment:
Out of scope but we should aim to move this to 5 (`5.6.2` I think?) as a follow-up 👀

Reply:
Thanks – I did this and everything still works 🎉